added angular icon in the source pane for angular projects

borian opened the pull request:
Associated Issue: #4733


### Summary of Changes

added angular icon in the source pane for angular projects

![light](https://user-images.githubusercontent.com/1525694/32992085-ae98ed28-cd46-11e7-9322-6a17fd220d74.png)
![dark](https://user-images.githubusercontent.com/1525694/32992086-b12a40c8-cd46-11e7-82f5-9c6db84bd648.png)

James300 commented on src/components/shared/Svg.css:
```diff
@@ -115,3 +115,14 @@ html .arrow.expanded svg {
   opacity: 0.5;
 }
+
+.managed-tree .tree-node .angular svg {
```
Any reason this isn't just `.angular`, I understand the difference but I'm curious if this is safer / more maintainable / something else?

borian replied:
Sorry, I only mentioned this in the issue https://github.com/devtools-html/debugger.html/issues/4733.
The icon is already used in the call stack view, thats why I added specific styling for the source pane

jasonLaster commented:
@borian can we replace `ng://` and `Angular`

borian commented:
@jasonLaster shure, I changed the name and squashed the commit
![ng-light](https://user-images.githubusercontent.com/1525694/33102301-69ee4ed4-cf1c-11e7-9363-2663085d53e7.png)
![ng-dark](https://user-images.githubusercontent.com/1525694/33102294-60f8054a-cf1c-11e7-8425-f179022023bc.png)

jasonLaster commented:
Thanks i made a couple of tweaks including the shorter classname, hopefully we'll be okay :)

borian commented:
This css will add additional styling to the icon used in the call stack view.
But it will only be 3px larger, i think 😄
